src/auditlog/middleware.py

```diff
@@ -41,7 +42,7 @@ def process_request(self, request):
             threadlocal.auditlog['remote_addr'] = request.META.get('HTTP_X_FORWARDED_FOR').split(',')[0]
 
         # Connect signal for automatic logging
-        if hasattr(request, 'user') and hasattr(request.user, 'is_authenticated') and request.user.is_authenticated():
+        if hasattr(request, 'user') and hasattr(request.user, 'is_authenticated') and is_authenticated(request.user):
```

Comment:
Maybe the attribute check can be combined with the compat function (and just return `False` if we don't know)? Just an idea, not sure if this would be practical.

Reply:
As above, have changed as suggested and all tested OK.

Consideration was to avoid adding logic to `compat.py` so that it can eventually be removed when <1.10 isn't supported.